canonical-websites/www.ubuntu.com
# Updated software defined network options on pricing page

## Done

- Updated software defined network options on pricing page
- Updated the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit)


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/pricing](http://0.0.0.0:8001/support/plans-and-pricing#openstack)
- compare to the copy doc

## Issue / Card

Fixes #2793
